week08_pomdp/practice_pytorch.ipynb

```diff
@@ -367,7 +386,7 @@
    "outputs": [],
    "source": [
     "# for each of n_parallel_games, take 10 steps\n",
-    "rollout_obs, rollout_actions, rollout_rewards, rollout_mask = pool.interact(10)"
+    "rollout_obs, rollout_actions, rollout_rewards, rollout_mask = pool.interact(25)"
```

Comment:
Any reason for this change?

Reply:
Testing purposes! I noticed that there are empty frames from 0 to 71 as I remember, which means that the less the interaction, the less the information we take, and since **policy gradient** is an ___on-policy___, that means it depends on the Monte Carlo data sampling of a specific random generation, I wanted to test out if I send more significant interactions, would that helps to have more trajectories. Hence, we would have a better policy or not.

The expectation Eτ∼πθ implies that as more trajectories τs are sampled using a policy πθ and averaged it
approaches the actual policy gradient rθJ(πθ) (Laura, Keng, 2020).